utils/update_checkout/update-checkout-config.json

```diff
@@ -39,6 +39,10 @@
         "pythonkit": {
             "remote": { "id": "pvieito/PythonKit" }
         },
+        "binutils": {
+            "remote": { "id": "bminor/binutils-gdb" },
```

Comment:
This is not the official repository, please dont use this.
CC: @shahmishal

Reply:
Before this PR, should I support non-github hosted repository for update-checkout?